package.json

```diff
@@ -18,6 +18,7 @@
     "nodemon": "^1.14.11",
     "probot": "^5.0.0",
     "probot-app-todos": "^1.0.5",
+    "probot-changelog": "github:mikz/probot-changelog#8441e070926211ad32b5f0430c9fe30a26f97c6d",
```

Comment:
That's a strangely specific, yet not labelled, version. Is it just current-tip-of-master? If so, how do we keep it up to date?

Reply:
Yeah it's just the current tip of master. They don't seem to have any releases so I just "pinned" it to what the version of master is at this point. That way greenkeeper.io (the JS version of pyupbot) _might_ be able to tell if newer versions exist.

Or, we could eventually ask the dev to start tagging versions, or re-implement it ourselves, or go with the other approach in the Issue...